README.md

```diff
@@ -20,18 +20,17 @@ That's why, contrary to most of the current online mobile emulators, the Mobile
 Mobile Checker is a Node application. It will eventually be distributed through npm, but in the meantime
 you can simply clone this repository:
 
-	git clone https://github.com/w3c/Mobile-Checker.git
+    git clone https://github.com/w3c/Mobile-Checker.git
 
 1. Install [Node.js](http://nodejs.org/)
 
 2. Install npm dependencies:
-````
-npm install -d
-```
+
+    npm install -d
```

Comment:
This still does not render correctly.
Could you please leave blank lines around the code block and indent it 3 or 4 spaces?
([An Example](https://gist.github.com/tripu/c594cf2873619a541807ed4cc955c580).)